types/voice-activity-detection/index.d.ts

```diff
@@ -0,0 +1,23 @@
+// Type definitions for voice-activity-detection 0.0
+// Project: https://github.com/Jam3/voice-activity-detection
+// Definitions by: JohnDoeAntler <https://github.com/JohnDoeAntler>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function vad(audioContext: AudioContext, stream: MediaStream, options?: VoiceActivityDetectionOptions): void;
+
+interface VoiceActivityDetectionOptions {
+    fftSize?: number;
+    bufferLen?: number;
+    smoothingTimeConstant?: number;
+    minCaptureFreq?: number;
+    maxCaptureFreq?: number;
+    noiseCaptureDuration?: number;
+    minNoiseLevel?: number;
+    maxNoiseLevel?: number;
+    avgNoiseMultiplier?: number;
+    onVoiceStart?: () => void;
+    onVoiceStop?: () => void;
+    onUpdate?: (val: number) => void;
+}
+
+export default vad;
```

Comment:
The [source](https://unpkg.com/voice-activity-detection@0.0.5/index.js) actually uses commonjs export, so you need to use the Typescript syntax for this:

`export = vad`

You'll also have to move `VoiceActivityDetectionOptions` inside a  `namespace vad` that will merge with `function vad`.